tests/nlu/selectors/test_selectors.py

```diff
@@ -66,3 +69,60 @@ def test_train_selector(pipeline, component_builder, tmpdir):
         assert rank.get("name") is not None
         assert rank.get("confidence") is not None
         assert rank.get("full_retrieval_intent") is not None
+
+
+def test_training_label():
+
+    # use data that include some responses
+    td = load_data("data/examples/rasa/demo-rasa.md")
```

Comment:
please use more readable names (`td`, `rs`, `r`). and please don't reassign them